### PR TITLE
Connect button, feedback name of web3 provider

### PR DESCRIPTION
**Pull request by dkent600**
Resolves:  #1010 
Resolves: https://daostack.tpondemand.com/RestUI/Board.aspx#page=board/5209716961861964288&appConfig=eyJhY2lkIjoiQjgzMTMzNDczNzlCMUI5QUE0RUE1NUVEOUQyQzdFNkIifQ==&boardPopup=userstory/1430/silent

### CODE REVIEW

File: src/layouts/Header.tsx [hunk 1]
```diff
@@ -84,6 +87,8 @@ class Header extends React.Component<IProps, null> {
 
     const daoAvatarAddress = dao ? dao.address : null;
     const accountIsEnabled = getAccountIsEnabled();
+    const web3ProviderInfo = getWeb3ProviderInfo();
+    // const cachedWeb3ProviderInfo = this.props.getCachedWeb3ProviderInfo();
```
Comment (tibetsprague) [inline src/layouts/Header.tsx]: so is this not actually used then?

Comment (dkent600) [reply src/layouts/Header.tsx]: oh, I left that in because I anticipated that it is going to be used soon, but doubtfully in this PR.  So I can remove it.

**Comment by dkent600** [comment]
> i still see "log in" and "log out" in the review app:

@jellegerbrandy by design per the TP ticket

**Comment by jellegerbrandy** [comment]
> > i still see "log in" and "log out" in the review app:
> 
> @jellegerbrandy by design per the TP ticket

Well mixing "log in" and "connect" metaphors is even more confusing then before, I think... @A-Zak , are you sure you want this?

**Pull request merged**